vuepress/docs/refarch/README.md

```diff
@@ -1,14 +1,362 @@
+**[Space for diagram/map]**
+
+
+### New and Unclassified (non-domain)
+
+
+
+<p id="gdcalert4" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image4.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert5">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+
+![alt_text](images/image4.png "image_tooltip")
+
+
+A number of New and Unclassified (non-domain) problems have been identified by (Business/Developers/Both Business & Developers).  Once Business and the System Architects have identified the required solution in order to solve the problem, they will be classified into one of the Problem containers and fulfilled in accordance with its processes.
+
+
+
+
+## Solution Space (_High level description and the context map)_
+
+The Solution Space defined by DDD-architecture is focused on how the business problems (improvements) identified in the Problem Space are going to be solved.  As a result it necessarily contains more information and technical details than the Problem Space.  It includes elements such as Ubiquitous Language, Bounded Contexts, and Cross-Cutting Concerns.
+
+
+
+<p id="gdcalert5" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image5.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert6">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+
+![alt_text](images/image5.png "image_tooltip")
+
+
+
+### Ubiquitous Language
+
+A challenge that most teams face is maintaining a clear understanding of terms that may not be unique with a particular Domain.  A classic example of a non-unique term is “account”: this term could refer to a set of financial accounts, entity profile, or a login name.
+
+As noted in the overview, Ubiquitous Language is used to aid in the elimination of confusion and miscommunication between business and technical teams working to solve a business problem or group of business problems.  Whilst it is possible that that each Domain/Subdomain may contain terms that are not unique, as noted above, within a particular context, and in the instance of DDD-architecture, that would be a Bounded Context, it is important to ensure that all terms are unique, clearly understood by all participants, and correctly applied.
+
+In the instance of Mojaloop, the naming convention for Ubiquitous Language terms is as follows:
+
+_(To be completed post discussion with the members of the Reference Architecture team on 07/19 or 07/20, 2021)_ 
+
+
+### Bounded Contexts
+
+The following Bounded Contexts have been identified and implemented in Mojaloop:
+
+(This is a high-level description of each Bounded Context.  A more detailed view follows later in this document)
+
+
+<table>
```

Comment:
Table to be converted to markdown --> https://www.markdownguide.org/extended-syntax/#tables

We can try use this tool to convert them --> https://jmalarcon.github.io/markdowntables/